src/nifake/metadata/functions_addon.py

```diff
@@ -69,6 +69,16 @@
     'EnumInputFunctionWithDefaults':    { 'parameters': { 1: { 'default_value': 'Turtle.LEONARDO', }, }, },
 }
 
+# Converted parameters
+functions_converters = {
+    'Read':                             { 'parameters': { 1: { 'python_api_converter_name': '_TimedeltaConverter', 
+                                                               'python_api_converter_type': 'datetime.timedelta',
+                                                               'python_api_converter_additional_params': ["'seconds'"], }, }, },
+    'ReadFromChannel':                  { 'parameters': { 2: { 'python_api_converter_name': '_TimedeltaConverter', 
+                                                               'python_api_converter_type': 'datetime.timedelta',
+                                                               'python_api_converter_additional_params': ["'microseconds'"], }, }, },
```

Comment:
we should avoid this by introducing more converters

Reply:
Done, but as functions.